### ISSUE TITLE
documentation for update/delete/create?

### ISSUE THREAD

**Issue by contra** [chute/turbine#3]
can't seem to find anything on this

**Comment by ntucker** [comment]
Turns out the author wrote the blog post about his idea, but hasn't completed the code yet.

If you look at the code you'll see there is nothing written for these things yet.

**Comment by petrbela** [comment]
Correct, updates are not finished yet, partially because the `mutation` operations in graphql have been a little unclear.
The focus in the recent days has been shifted more towards graphql-rest adapter that would work with Relay instead of building a full-featured data management library copying the work that the Relay team is doing.
As part of the prototype though, we might port the mutation operations to Turbine, too.
Also see #2.
